### ISSUE TITLE
Stochastic block model example/tutorial

### ISSUE THREAD

**Issue by dustinvtran** [tensorflow/probability#71]
See also https://github.com/blei-lab/edward/blob/master/examples/stochastic_block_model.py.

**Comment by rlouf** [comment]
I come from the network theory crowd and am keen to learn TFP (currently using PyMC3). Should this go in examples/jupyter_notebooks?

**Comment by dustinvtran** [comment]
Yeah, if you want to make it a notebook. Alternatively, examples/ if it's just a script similar to the Edward one.

**Comment by rlouf** [comment]
Ok, I'm on it!

**Comment by rlouf** [comment]
I'm at a roadblock: how is the probability mass function (PointMass in Edward) implement in tfp? I browsed the codebase (mostly generated_random_variables) but could not find it anywhere.

**Comment by srvasude** [comment]
Deterministic/VectorDeterministic should be the equivalent of PointMass (the former univariate, the latter multivariate).
